.github/workflows/build.yml

```diff
@@ -9,13 +9,19 @@ jobs:
       fail-fast: false
       matrix:
         os: [windows-2019, macos-11]
+        maven: ['3.8.4']
 
     runs-on: ${{ matrix.os }}
     timeout-minutes: 20
 
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      
+      - name: Set up Maven
+        uses: stCarolas/setup-maven@v4.3
```

Comment:
This is an action from a market place and I am not sure we should use it just like that. @eneufeld What is our policy regarding non-verified Github actions? Alternatively, we may need to do the setup ourselves in a custom step.

Reply:
I just checked what's else out there, maybe we could use the [Apache Maven Wrapper Plugin](https://maven.apache.org/wrapper/maven-wrapper-plugin/) for that as well. I will try that.